launcher/Application.cpp

```diff
@@ -371,7 +371,25 @@ Application::Application(int& argc, char** argv) : QApplication(argc, argv)
     }
 
     QString origcwdPath = QDir::currentPath();
+#if defined(Q_OS_LINUX)
+    const QString binFilePath = applicationFilePath();
+    const bool isAppImage = binFilePath.startsWith("/tmp/.mount_");
+    // Yes, this can technically trigger the logic below if someone makes an AppImage with an actual launcher exe named "ld-linux"
+    // Please don't :)
+    const bool executedFromLinker = QFileInfo(binFilePath).fileName().startsWith("ld-linux");
+
+    // NOTE(@getchoo): In order for `go-appimage` to generate self-contained AppImages, it executes apps from a bundled linker at
+    // <root>/lib64
+    // This is not the path to our actual binary, which we want
+    QString binPath;
+    if (isAppImage && executedFromLinker) {
+        binPath = FS::PathCombine(binFilePath, "../bin");
```

Comment:
```suggestion
        binPath = FS::PathCombine(binFilePath, "../../bin");
```
<img width="1129" height="53" alt="image" src="https://github.com/user-attachments/assets/9405cf40-6478-455d-93a9-728e6e17b050" />